### PR TITLE
149126865 replay exp

### PR DESCRIPTION
**Pull request by popaulina**

### CODE REVIEW

**Comment by mach-kernel** [comment]
Is there backend logic for this?

**Comment by popaulina** [comment]
As of now they're just columns

File: lib/api/entities/experiment.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Experiment < Base
       property :id
       property :name
       property :repeats
```
Comment (mach-kernel) [inline lib/api/entities/experiment.rb]: This still a field?

Comment (popaulina) [inline lib/api/entities/experiment.rb]: yes

**Pull request merged**